From 55b158c0fe758817982ac2055aced3bee398e746 Mon Sep 17 00:00:00 2001
From: Lawrence Rust <lvr@softsystem.co.uk>
Date: Mon, 16 May 2011 21:17:39 +0200
Subject: [PATCH 034/285] netvision: autoplay BBC iPlayer streams

Change bbciplayer.html so that a mouse click isn't needed to start playback.
This avoids the problem with emulated mouse clicks from the numeric keypad
not being recognized by Flash player.
---
 .../nv_python_libs/configs/HTML/bbciplayer.html    |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/mythtv/programs/scripts/internetcontent/nv_python_libs/configs/HTML/bbciplayer.html b/mythtv/programs/scripts/internetcontent/nv_python_libs/configs/HTML/bbciplayer.html
index fafc015..cca7ad1 100644
--- a/mythtv/programs/scripts/internetcontent/nv_python_libs/configs/HTML/bbciplayer.html
+++ b/mythtv/programs/scripts/internetcontent/nv_python_libs/configs/HTML/bbciplayer.html
@@ -1,7 +1,8 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
 <html>
   <head>
-  <!-- Author: R.D. Vaughan       Apr 28th, 2010
+  <!-- Author: R.D. Vaughan
+       Apr 28th, 2010
        Purpose: Implement full screen browser video display for the BBC iPlayer
   Example:
   file:///usr/local/share/mythtv/mythnetvision/scripts/nv_python_libs/configs/HTML/bbciplayer.html?videocode=b00s5nfv
@@ -20,7 +21,7 @@
         }
 
       var videocode = gup( 'videocode' );
-      var embedded = '<embed width="100%" height="98%" allowfullscreen="true" allowscriptaccess="always" wmode="default" quality="high" flashvars="embedReferer=&amp;embedPageUrl=http%3A%2F%2Fwww.bbc.co.uk%2Fiplay&#10;er%2Fepisode%2FVIDEOCODE%2F%3F&#10;t%3D2m53s&amp;domId=bip-play-emp&amp;config=http%3A%2F%2Fwww.bbc.co.uk%2Femp&#10;%2Fiplayer%2Fconfig.xml&amp;playlist=http%3A%2F%2Fwww.bbc.co.uk%2Fiplayer%2F&#10;playlist%2FVIDEOCODE&amp;holdingImage=http%3A%2F%2Fnode1.bbcimg.co.uk%2Fiplay&#10;er%2Fimages%2Fepisode%2FVIDEOCODE_640_360.jpg&amp;config_settings_bitrateFloo&#10;r=0&amp;config_settings_bitrateCeiling=2500&amp;config_settings_transportHei&#10;ght=35&amp;config_settings_cueItem=b00rqkg0%3A173&amp;config_settings_autoPl&#10;ay=true&amp;config_settings_showPopoutCta=false&amp;config_messages_diagnost&#10;icsMessageBody=Insufficient%20bandwidth%20to%20stream%20this%20programme.%20&#10;Try%20downloading%20instead%2C%20or%20see%20our%20diagnostics%20page.&amp;co&#10;nfig_settings_language=en&amp;guidance=unset" id="bbc_emp_embed_bip-play-emp" src="http://www.bbc.co.uk/emp/10player.swf?revision=15501_15796" type="application/x-shockwave-flash"/>';
+      var embedded = '<embed width="100%" height="95%" allowfullscreen="true" allowscriptaccess="always" wmode="default" quality="high" flashvars="embedReferer=&amp;embedPageUrl=http%3A%2F%2Fwww.bbc.co.uk%2Fiplayer%2Fepisode%2FVIDEOCODE%2F&amp;domId=bip-play-emp&amp;config=http%3A%2F%2Fwww.bbc.co.uk%2Femp%2Fiplayer%2Fconfig.xml&amp;playlist=http%3A%2F%2Fwww.bbc.co.uk%2Fiplayer%2Fplaylist%2FVIDEOCODE&amp;holdingImage=http%3A%2F%2Fnode1.bbcimg.co.uk%2Fiplayer%2Fimages%2Fepisode%2FVIDEOCODE_640_360.jpg&amp;config_settings_bitrateFloor=0&amp;config_settings_bitrateCeiling=2500&amp;config_settings_transportHeight=35&amp;config_settings_autoPlay=true&amp;config_settings_showPopoutCta=false&amp;config_settings_showPopoutButton=false&amp;config_messages_diagnosticsMessageBody=Insufficient%20bandwidth%20to%20stream%20this%20programme.%20Try%20downloading%20instead%2C%20or%20see%20our%20diagnostics%20page.&amp;config_settings_language=en&amp;guidance=unset" id="bbc_emp_embed_bip-play-emp" src="http://www.bbc.co.uk/emp/10player.swf" type="application/x-shockwave-flash" autostart="true"/>';
       document.write('<title>BBC iPlayer Full Screen</title>');
       document.write(embedded.replace(/VIDEOCODE/g, videocode));
 
-- 
1.7.9.5

